### ISSUE TITLE
Downloadliste/Download ergänzt um Ansicht "Downloadliste/Download ohne Datei-Informationen"

### ISSUE THREAD

**Issue by gs-uvek** [swiss/styleguide#508]
Derzeit können mit der Komponente Download und Downloadliste nur Listen mit Titel inkl. den Dateiinformationen (Dateityp, Grösse, Datum) erzeigt werden. Neu soll die Option "Datei-Informationen ausblenden" möglich sein.

**Comment by bit-pro-iew-eui** [comment]
@gs-uvek in the webguideline we do not have a "Download" component. Is this functionality in your CMS Adobe Experience Manager?

**Comment by gs-uvek** [comment]
Ja, die Download-Komponente dient der Darstellung von Dateien, z.B. pdfs:
![download-komponente](https://cloud.githubusercontent.com/assets/25457594/22743352/3e5da546-ee1a-11e6-9b4e-85be406ade39.JPG)
Laut CD Bund muss aber immer Dateityp, Grösse und Datum mit dem Titel angezeigt werden?

**Comment by bit-pro-iew-eui** [comment]
ok, question understood, answer pending.

**Comment by bit-pro-iew-eui** [comment]
Bitte Begründung angeben, wieso sollen diese Attributen ausgeblendet werden?

**Comment by gs-uvek** [comment]
Aus Darstellungsgründen, aber die Nachfrage nach einer Lösung ist nicht mehr gross, so dass dieses Issue geschlossen werden kann.
